README.md

```diff
@@ -245,7 +244,49 @@ config :coherence,
   max_failed_login_attempts: 3
 ```
 
-## Phoenix Channel Authentication
+### Custom registration and sessions routes
+
+Coherence supports custom routes for registration and login. These configurations can be set globally or scoped.
+
+Which routes can be custom?
+
+```
+%{
+  registrations_new:  "/registrations/new",
+  registrations:      "/registrations",
+  passwords:          "/passwords",
+  confirmations:      "/confirmations",
+  unlocks:            "/unlocks",
+  invitations:        "/invitations",
+  invitations_create: "/invitations/create",
+  invitations_resend: "/invitations/:id/resend",
+  sessions:           "/sessions",
+  registrations_edit: "/registrations/edit"
+}
+```
+
+To set them globally add the following to you configuration:
+
+
+
+```elixir
+config :coherence,
+  default_routes: %{
+    registrations_edit: "/accounts/edit", ...},
+```
+
+To set them scoped for each mode (protected, public, etc..):
+
+```elixir
+scope "/" do
+  pipe_through :protected
+  coherence_routes :protected, [
+    custom_routes: %{registratons_edit: "/accounts/edit", ...}
+  ]
+end
+``
+
+# Phoenix Channel Authentication
```

Comment:
I think this should be kept as a second-level title.

Reply:
True, my bad